js/projectView.js

```diff
@@ -1,3 +1,4 @@
+(function(module) {
 $(document).ready(function() {
```

Comment:
You could probably move all this document.ready stuff to the initIndexPage function now you have it.